Exception in list: 'AnonymousUser' object has no attribute 'user_id'
User: Anonymous
Session: {'session_key': '2ztx6huc3cohqlk5lhk6wlu54p2xojcl', 'session_expiry': datetime.datetime(2025, 8, 19, 20, 52, 25, 638894, tzinfo=datetime.timezone.utc), 'session_data': {'session_token': 'LRN5SRzDfRirICwiobjYFUNviphiMBoE'}}
Path: /order-api/order-detail-api/
Method: GET
Data: {}
Traceback (most recent call last):
  File "C:\Users\Divyam Shah\OneDrive\Desktop\Dynamic Labz\Clients\Clients\Oven fresh\OvenFresh\ovenfresh_ecommerce\utils\decorators.py", line 47, in _wrapped_view
    return view_func(self, request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\OneDrive\Desktop\Dynamic Labz\Clients\Clients\Oven fresh\OvenFresh\ovenfresh_ecommerce\Order\views.py", line 594, in list
    user_id = request.user.user_id
              ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'AnonymousUser' object has no attribute 'user_id'
Internal Server Error: /order-api/order-detail-api/
"GET /order-api/order-detail-api/?order_id=8561091640 HTTP/1.1" 500 323
Exception in list: 'AnonymousUser' object has no attribute 'user_id'
User: Anonymous
Session: {'session_key': '2ztx6huc3cohqlk5lhk6wlu54p2xojcl', 'session_expiry': datetime.datetime(2025, 8, 19, 20, 52, 27, 267022, tzinfo=datetime.timezone.utc), 'session_data': {'session_token': 'LRN5SRzDfRirICwiobjYFUNviphiMBoE'}}
Path: /order-api/order-detail-api/
Method: GET
Data: {}
Traceback (most recent call last):
  File "C:\Users\Divyam Shah\OneDrive\Desktop\Dynamic Labz\Clients\Clients\Oven fresh\OvenFresh\ovenfresh_ecommerce\utils\decorators.py", line 47, in _wrapped_view
    return view_func(self, request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\OneDrive\Desktop\Dynamic Labz\Clients\Clients\Oven fresh\OvenFresh\ovenfresh_ecommerce\Order\views.py", line 594, in list
    user_id = request.user.user_id
              ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'AnonymousUser' object has no attribute 'user_id'
Internal Server Error: /order-api/order-detail-api/
"GET /order-api/order-detail-api/?order_id=8561091640 HTTP/1.1" 500 323
Exception in retrieve: Could not parse the remainder: '[0]' from 'product_obj.photos[0]'
User: Divyam (AD0648349366)
Session: {'session_key': 'dey9xky5loudm179h8hyqhkb3w0qv7hy', 'session_expiry': datetime.datetime(2025, 9, 4, 16, 41, 43, 958220, tzinfo=datetime.timezone.utc), 'session_data': {'_auth_user_id': '1', '_auth_user_backend': 'django.contrib.auth.backends.ModelBackend', '_auth_user_hash': '4fb25e39b7f30d6edbc9f1e2ac8569e4e6771c4d2e3285c71d14e1c6e22b35f0'}}
Path: /product/dutch-truffle-shine-cake-eggless/
Method: GET
Data: {}
Traceback (most recent call last):
  File "C:\Users\Divyam Shah\OneDrive\Desktop\Dynamic Labz\Clients\Clients\Oven fresh\OvenFresh\ovenfresh_ecommerce\utils\decorators.py", line 47, in _wrapped_view
    return view_func(self, request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\OneDrive\Desktop\Dynamic Labz\Clients\Clients\Oven fresh\OvenFresh\ovenfresh_ecommerce\FrontEnd\views.py", line 187, in retrieve
    return render(request, 'product-detail.html', data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
           ^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
               ^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 489, in parse
    raise self.error(token, e)
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 487, in parse
    filter_expression = self.compile_filter(token.contents)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 605, in compile_filter
    return FilterExpression(token, self)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Divyam Shah\AppData\Roaming\Python\Python312\site-packages\django\template\base.py", line 706, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '[0]' from 'product_obj.photos[0]'
Internal Server Error: /product/dutch-truffle-shine-cake-eggless/
"GET /product/dutch-truffle-shine-cake-eggless/ HTTP/1.1" 500 6463
